### PR TITLE
[stable/mariadb] expose all parameters in values.yaml for mariadb.

### PR DESCRIPTION
**Pull request by gyliu513**
Fixed part of https://github.com/kubernetes/charts/issues/2116

/cc @prydonius

### CODE REVIEW

File: stable/mariadb/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 2.0.0
+version: 2.0.1
```
Comment (javsalgar) [inline stable/mariadb/Chart.yaml]: The version is outdated now. I would rebase with the latest changes

**Review by javsalgar — CHANGES_REQUESTED** [review]

**Comment by sameersbn** [comment]
@javsalgar i've rebased the chart on behalf of @gyliu513 
/lgtm

**Comment by gyliu513** [comment]
Thanks @sameersbn !

**Comment by javsalgar** [comment]
Great, now LGTM.

/lgtm
/ok-to-test

**Comment by mattfarina** [comment]
/retest

Note, there were changes to the underling infra between the last test run and this one. Issues had come up with unschedulable pods that I think are now fixed.

**Comment by javsalgar** [comment]
@gyliu513 any idea about these failures? It doesn't look to me that it should fail. Is it an infrastructure issue?

**Comment by prydonius** [comment]
Are we sure uncommenting any of these values hasn't broken anything?

**Comment by javsalgar** [comment]
In my minikube system, it stays infinitely in Init if this part is uncommented:

```
storageClass: "-"
```

If I comment that out it works.

File: stable/mariadb/values.yaml [hunk 2]
```diff
@@ -64,16 +64,16 @@ persistence:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  # storageClass: "-"
+  storageClass: "-"
```
Comment (prydonius) [inline stable/mariadb/values.yaml]: this should be `storageClass: ` without `"-"` as the default. Setting `"-"` will disable dynamic provisioning which is causing the test to fail.

**Comment by prydonius** [comment]
@javsalgar ah of course, that makes sense. That value should just be empty.

File: stable/mariadb/values.yaml (hunk 2 above)
Comment (gyliu513) [reply stable/mariadb/values.yaml]: Done, updating it to empty

**Comment by javsalgar** [comment]
@prydonius now lgtm

File: stable/mariadb/values.yaml [hunk 3]
```diff
@@ -64,16 +64,16 @@ persistence:
...
 
 ## Configure MariaDB with a custom my.cnf file
 ## ref: https://mariadb.com/kb/en/mariadb/configuring-mariadb-with-mycnf/#example-of-configuration-file
 ##
-# config: |-
-#   [mysqld]
-#   innodb_buffer_pool_size=2G
+config: |-
+  [mysqld]
+  innodb_buffer_pool_size=2G
```
Comment (prydonius) [inline stable/mariadb/values.yaml]: This should also be empty as it's just an example

**Pull request merged**